stdsimd-verify/build.rs

```diff
@@ -0,0 +1,24 @@
+use std::path::Path;
+
+fn main() {
+    let dir = Path::new(env!("CARGO_MANIFEST_DIR"));
+    let root = dir.parent().unwrap();
+    let root = root.join("coresimd/src/x86");
+    walk(&root);
+}
+
+fn walk(root: &Path) {
+    for file in root.read_dir().unwrap() {
+        let file = file.unwrap();
+        if file.file_type().unwrap().is_dir() {
+            walk(&file.path());
+            continue
+        }
+        let path = file.path();
+        if path.extension().and_then(|s| s.to_str()) != Some("rs") {
+            continue
+        }
+
+        println!("cargo:rerun-if-changed={}", path.display());
```

Comment:
Does this only redo the build.rs if any of the selected paths changed? What happens if I add a new path (e.g. by creating a new file in the `i686`/`x86_64` directories? Does it get automatically checked?

Reply:
Yes, this only reruns if those are changed. If you add a new module it doesn't matter, what typically happens is you add `mod foo` to an existing file.

Reply:
Cool! TIL that `build.rs` could do this. Every crate I have with a `build.rs` file gets rebuild all the time, maybe one should add something like this to the jemalloc-sys crate as well.